apps/api/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/api/App.kt

```diff
@@ -58,22 +58,31 @@ object Routes {
 
 fun main() {
     val rapid = RapidApplication.create(System.getenv())
+    val aivenRedisConnection =
+        RedisConnection(
+            host = Env.Redis.host,
+            port = Env.Redis.port,
+            username = Env.Redis.username,
+            password = Env.Redis.password,
+        )
     val redisConnection = RedisConnection(Env.Redis.url)
 
     embeddedServer(
         factory = Netty,
         port = 8080,
-        module = { apiModule(rapid, redisConnection) },
+        module = { apiModule(rapid, aivenRedisConnection, redisConnection) },
```

Comment:
Hvorfor to ulike redis-instanser?

Reply:
Kun for å teste. Jeg trodde jeg hadde satt denne PR-en til draft.